software/lazydogpile.livecodescript

```diff
@@ -0,0 +1,954 @@
+﻿script "lazydogpile"
+/*
+This stack is mostly an empty "hull" that ties everything else together
+Some stuff, like the Livecode engine and objects, need to be binary
+Everything else is in text files
+*/
+local sMain -- main[doc][rec][tag]=data/pointers
+-- logic
+local sPointerTags -- space delimited list of pointer categories
+local sMainShort
+local sTopoSort
+local sNoteList
+local sNotVisited
+--  display
+local sNowDoc -- the document the user is currently focused on ! requires update
+local sNowItem -- cr delimited list of records currently selected ! requires update
+local sNowFocus -- ! from GUI test
+local sGraphDrawing -- ! from GUI test; sGraphData + layer info
```

Comment:
@matthewmaier One extra variable naming convention you might like to consider is for arrays adding a capital A at the end so `sGraphDrawing` becomes `sGraphDrawingA`